tlatools/org.lamport.tlatools/src/pcal/Tokenize.java

```diff
@@ -935,6 +939,13 @@ else if (Misc.IsLetter(nextChar))
                     { addNextChar();
                       state = ID;  
                     }
+                  else if ('ℕ' == nextChar

```

Comment:
I was just curious for my own benefit of understanding the code base here; I noticed that these are handled uniquely vs any other char.  What is it about handling these that's different?

Reply:
These individual codepoints refer to the Nat, Int, and Real sets. We don't want them to be allowed as valid codepoints inside general identifiers.

Reply:
The following three lines are causing a compiler error on @xxyzzn's Windows machine. In the Eclipse editor, the Unicode characters appear as garbled text instead of being displayed correctly. When attempting to save the file in the Eclipse editor, the following error is triggered:

![image](https://github.com/user-attachments/assets/8c19f38d-4604-42db-b49d-73fdb427a8b4)

Reply:
I will boot into windows to see whether I can replicate the issue

Reply:
Probably Win11

Reply:
Can you check? This is a windows codepage issue. For some reason his system isn't accepting the file as being encoded with utf-8 and is trying to interpret it as Windows-1252, which is a windows-specific text encoding (see https://en.wikipedia.org/wiki/Windows-1252). Since `Tokenize.java` is encoded in utf-8 (verified with the `file` command in linux and macOS`), this would seem to indicate that his system isn't set up to support UTF-8.

Reply:
It could also be helpful for him to run this command:
```
REG QUERY "HKLM\SYSTEM\CurrentControlSet\Control\Nls\CodePage" | findstr /I "CP.*REG_SZ"
```

Which should output something like (numbers may vary):
```
    ACP    REG_SZ    1252
    OEMCP    REG_SZ    437
    MACCP    REG_SZ    10000
```